packages/react-img-viewer/src/ImageViewer.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useRef, useEffect } from 'react';
+import { FullScreenViewer, ImageViewer } from 'iv-viewer';
+import 'iv-viewer/dist/iv-viewer.css';
+import { ImageViewerProps } from './types';
+
+export const NormalViewer: React.FC<ImageViewerProps> = ({
+  img,
+  hiResImg,
+  width,
+  height,
+  zoom,
+  defaultZoom = 100,
+  maxZoom = 500,
+  snapView = true,
+  refreshOnResize = true,
+  zoomOnMouseWheel = true,
+  hasZoomButtons = true,
+  zoomStep,
+  onZoomChange,
+  onLoad,
+}) => {
+  const viewerRef = useRef<any>(null);
```

Comment:
As we are storing viewer instance on the ref, lets define the proper type. 
```suggestion
  const viewerRef = useRef<BaseImageViewer | null>(null);
```